convene-web/app/models/space.rb

```diff
@@ -56,4 +53,22 @@ def unlocked?
   def locked?
     access_level&.to_sym == :locked
   end
+
+  # @see {Utilities}
+  # @see {UtilityHookup}
+  # @returns {ActiveRecord::Relation<UtilityHookups>}
+  has_many :utility_hookups
+
+  def jitsi_meet_domain
+    jitsi_hookup = utility_hookups.find_by(utility_slug: :jitsi)
+    return if jitsi_hookup.blank?
+
+    Utilities.from_utility_hookup(jitsi_hookup).meet_domain
+  end
+
+  def hookups
```

Comment:
Is this used anywhere? I couldn't find a usage example in this PR (but its big, so likely I missed it?). Looking at the implementation, it reads like it returns Utilities, not UtilityHookups, so I was trying to understand how it is used.

Reply:
Probably not! Thank you!